.github/workflows/tests.yml

```diff
@@ -0,0 +1,73 @@
+name: tests
```

Comment:
Right now GitHub shows me this when checking the tasks which is a little confusing and not describing whats actually running:

```
@github-actions
tests / rust build (push)

@github-actions
tests / js build (push)
```

Maybe name this `Build and test` and the jobs `p2panda-rs` and `p2panda-js` instead of `rust build` and `js build`? Then it would read:

```
Build and test / p2panda-rs
Build and test / p2panda-js
```